package.xml

```diff
@@ -3,14 +3,22 @@
   <name>psen_scan_v2</name>
   <version>0.10.1</version>
   <description>ROS support for the Pilz laser scanner</description>
-
+  
   <maintainer email="a.gutenkunst@pilz.de">Alexander Gutenkunst</maintainer>
   <maintainer email="c.doehn@pilz.de">Christian Döhn</maintainer>
   <maintainer email="c.henkel@pilz.de">Christian Henkel</maintainer>
   <maintainer email="i.martini@pilz.de">Immanuel Martini</maintainer>
   <maintainer email="r.feistenauer@pilz.de">Richard Feistenauer</maintainer>
 
   <license>LGPLv3</license>
+  
+  <url type="website">https://wiki.ros.org/psen_scan_v2</url>
```

Comment:
As far as I see this will display as "External Website", but will lead to the site you are already on. I would skip it.

Reply:
Or link to https://www.pilz.com/en-INT/products/robotics/ros-modules ?